debezium-core/src/main/java/io/debezium/converters/CloudEventsConverter.java

```diff
@@ -431,6 +431,9 @@ else if (schema.type() == Type.INT8 || schema.type() == Type.INT16 || schema.typ
         else if (schema.type() == Type.STRING || schema.type() == Type.INT64) {
             ceExtensionSchema = SchemaBuilder.string();
         }
+        else if (schema.type() == Type.STRUCT) {
```

Comment:
Have you tested this? I'd expect that the nested attributes will need proper handling, i.e. prefixing them with with some constant literal as they are all put into one flat namespace. That said I'm a bit dubious about the nested structure begin with. This ties into my question about the purpose of these attributes in change events themselves. I kinda thought we'd _only_ need the TX id, in which case the question of nesting is a moot point.

Reply:
Yes, CE test fails due to this reason, thus I added this as a hotfix - for whether we need or not other information please see other comment.

Reply:
Removed as transaction is no longer in source info block